recipe/patches/0001-always-build-quarto-preview-js.patch

```diff
@@ -7,14 +7,15 @@ to remove this patch. See:
 https://github.com/conda-forge/quarto-feedstock/pull/36#issuecomment-1947515694 
 
 diff --git a/src/webui/quarto-preview/build.ts b/src/webui/quarto-preview/build.ts
-index 625cc7910..728afb060 100644
+index 55d4e4b6b..44c3a8001 100644
 --- a/src/webui/quarto-preview/build.ts
 +++ b/src/webui/quarto-preview/build.ts
-@@ -11,25 +11,7 @@ try {
+@@ -12,26 +12,7 @@ try {
+   jsBuildTime = 0;
  }
  
- // check if any of our repo files have a later time
--let build = false;
+-// check if any of our repo files have a later time
+-let build = args.length > 0 && args.includes("--force");
```

Comment:
I feel I should somehow be leveraging this mechanism to force the build, but didn't have time to figure out whether / how I can influence that from our build script.

Reply:
>So as you are calling prepare-dist already it should build. But the recipe is also calling configure first, so it builds a first time too
>
> ...
>
> Maybe this patch is not needed anymore ?

Thanks so much for helping with this context, Christophe! It's often easy to lose track of these things between releases!

I have a long holiday weekend for US independence day and I'll give this a test.